src/control/bitraversable/instances.lean

```diff
@@ -76,7 +76,7 @@ instance bitraversable.flip : bitraversable (flip t) :=
 open is_lawful_bitraversable
 instance is_lawful_bitraversable.flip [is_lawful_bitraversable t]
   : is_lawful_bitraversable (flip t)  :=
-by constructor; intros; unfreezingI { casesm is_lawful_bitraversable t }; apply_assumption
+by constructor; intros; unfreezingI { casesm is_lawful_bitraversable t }; tactic.apply_assumption
```

Comment:
Somewhat confusingly we presently have two tactics `tactic.apply_assumption` and `tactic.interactive.apply_assumption`. They are not exactly the same. Either works here, but it was only an accident that this proof switched to using the interactive one... So I've changed it back!

I'll try to deduplicate these tactics separately.